Makefile

```diff
@@ -652,9 +654,10 @@ deploy: manifests kustomize
 
 # Generate manifests e.g. CRD
 # Can also generate RBAC and webhooks but that is not enabled currently
-manifests: controller-gen
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./api/..." output:crd:artifacts:config=config/crd/bases
-	for x in $$(find config/crd/bases/*); do sed -i -e '/creationTimestamp: null/d' $$x; done
+manifests:
+	$(DOCKER_RUN) sh -c "\
+		controller-gen $(CRD_OPTIONS) paths="./api/..." output:crd:artifacts:config=config/crd/bases \
+		for x in $$(find config/crd/bases/*); do sed -i -e '/creationTimestamp: null/d' $$x; done"
```

Comment:
I've been trying the Makefile's changes locally and had trouble with this when running `make gen-files` (or `make manifests` for that matter). This ended up working for me:
```
	$(DOCKER_RUN) sh -c "\
		controller-gen $(CRD_OPTIONS) paths='./api/...' output:crd:artifacts:config=config/crd/bases;\
		find config/crd/bases/* | xargs sed -i -e '/creationTimestamp: null/d'"
```

(basically I added a semi colon to the end of the first line and used xargs instead of a for loop)